lib/jxl/enc_color_management.cc

```diff
@@ -244,10 +253,13 @@ Status DoColorSpaceTransform(void* cms_data, const size_t thread,
     }  // else: in-place, no need to copy
   } else {
 #if JPEGXL_ENABLE_SKCMS
-    JXL_CHECK(skcms_Transform(
-        xform_src, skcms_PixelFormat_RGB_fff, skcms_AlphaFormat_Opaque,
-        &t->profile_src, buf_dst, skcms_PixelFormat_RGB_fff,
-        skcms_AlphaFormat_Opaque, &t->profile_dst, xsize));
+    JXL_CHECK(
+        skcms_Transform(xform_src,
+                        (t->channels_src != 4 ? skcms_PixelFormat_RGB_fff
```

Comment:
This is quite minor but I feel that perhaps this would be more legible with the condition inverted.

Reply:
updated.